apps/extension/webpack.config.js

```diff
@@ -115,10 +115,8 @@ export default (env, argv) => {
         patterns: [
           { from: 'public', to: '.' },
           {
-            from: 'bin',
-            to: 'bin',
-            context: path.resolve(__dirname, '../../packages/wasm'),
-            noErrorOnMissing: true,
+            from: process.env.KEY_PATH,
+            to: 'keys',
```

Comment:
perhaps import.meta.resolve instead of env

Reply:
.